src/views/DogEdit/DogEdit.test.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import DogEdit from './DogEdit';
+
+test('<DogEdit/> should render the specific dog form information', async () => {
+  const { container } = render(
+    <MemoryRouter initialEntries={['/dogs/5/edit']}>
+      <Route path="/dogs/:id/edit">
+        <DogEdit />
+      </Route>
+    </MemoryRouter>
+  );
+
+  await screen.findByText('Image:');
```

Comment:
`findByDisplayValue(something that comes from the actual dog being edited)` to avoid a memory leak and properly test the component